srcpkgs/cryptomator-cli/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'cryptomator-cli'
+pkgname=cryptomator-cli
+version=0.4.0
+revision=1
+wrksrc="cli-${version}"
+makedepends="apache-maven"
+depends="openjdk11 davfs2"
```

Comment:
Is it really openjdk11 or does it work with an earlier one?

Reply:
The readme says 'at least JDK 11'. I tried running it with JDK 8 but it is compiled for JDK 11.